src/components/tab-group/tab-panel/bl-tab-panel.ts

```diff
@@ -40,7 +40,7 @@ export default class BlTabPanel extends LitElement {
    * This attribute set by `tab-group` to make panel visible or hidden.
    */
   @property({ type: Boolean, reflect: true })
-  visible = false;
+  hidden = true;
```

Comment:
@muratcorlu `hidden` property is more suitable here according to a11y standards, more info: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#example

Reply:
Looks good to me.